src/mixins/userIsTabbing.js

```diff
@@ -0,0 +1,22 @@
+const userIsTabbingMixin = {
+	created: function() {
+		window.addEventListener("keydown", this.handleFirstTab);
+		this.$userIsTabbing = false;
+	},
+	methods: {
+		handleFirstTab: function(e) {
+			if (e.key === "Tab" || e.keyCode.toString() === "9") {
+				window.removeEventListener("keydown", this.handleFirstTab);
+				window.addEventListener("click", this.handleFirstClick);
+				this.$userIsTabbing = true;
+			}
+		},
+		handleFirstClick: function() {
```

Comment:
```suggestion
		$_handleFirstClick: function(e) {
```
to prevent namespace conflicts
https://vuepress.test.schul-cloud.org/0-GettingStarted/3-DirectoryStructure.html#src-mixins